### ISSUE TITLE
Add new E1M5 music?

### ISSUE THREAD

**Issue by Voros2** [freedoom/freedoom#499]
http://s000.tinyupload.com/index.php?file_id=84107328017836047279

This is a cleaned up version of a track jupiter_ex had done before. It was decided that it would go into slot e1m5, since e1m5 and e2m6 share the same track.

How about adding it now?

**Comment by ElonSatoshi** [comment]
Put it in a pull request Contributing to Freedoom is a good way to learn how to use Git.

I can't listen to the music, it is showing as a blank web page.

**Comment by Voros2** [comment]
I would, but I can't right now. That's why I'm suggesting it here, so that someone else can see it or the maintainers can directly add it the repo themselves.

**Comment by Voros2** [comment]
Here's a more stable link: http://www.geocities.ws/v/o/voros/musics/freedoom_v2_t0.mid

**Comment by Litrivin** [comment]
Files differ, the second one seems truncated:

http://s000.tinyupload.com/index.php?file_id=84107328017836047279
```
freedoom_v2_t0.mid
Size: 15.99KB (16370B)
MD5: 545d2331213870664183f6884bd7c5b
```

http://www.geocities.ws/v/o/voros/musics/freedoom_v2_t0.mid
```
freedoom_v2_t0.mid
Size: 8.57KB (8771B)
MD5: 9d590e7e9b4f8ce5fda2f55682add3bc
```
Also, I think this MIDI is way too loud and its volume should be normalized.

**Comment by Proluiz** [comment]
Voros link is missing it's possible to you guys repost the original version, otherwise it's lost...

**Comment by Xindage** [comment]
closed, by #730
